src/core/wallet.rs

```diff
@@ -0,0 +1,34 @@
+use crate::core::crypto::{Crypto};
+
+pub struct OwnedCrypto {
+  pub name: String,
+  pub volume: f32,
+  pub avg_price: f32,
+  pub cover: i32, //number of cover
+  pub source_balance: f32
+}
+
+pub struct Wallet {
+  pub user_alias: String
+}
+
+impl Wallet {
+  pub fn get_holding_item(&self, crypto:&Crypto) -> Option<OwnedCrypto> {
+    println!("get_holding_item: {}",crypto);
+    return None;
```

Comment:
```suggestion
    println!("get_holding_item: {}", crypto);

    return None;
```

Should we add a line before a `return` statement?